test/activerecord_spanner_adapter/connection_test.rb

```diff
@@ -45,6 +45,7 @@ def test_get_database
   end
 
   def test_connection_is_active
+    connection.connect!
```

Comment:
The connection should not be active before it actually has connected. The `active?` method should not alter the state of the connection.